feat: Separate Goals tab in repository Settings PLUTO-532 (#1681)
codacy/docs: docs/repositories-configure/adjusting-quality-goals.md

> # Adjusting goals for code quality
> 
> You can adjust the **Goals** that help you monitor the progress of the code quality in your repository dashboard, and configure which files Codacy considers complex or duplicated.

Reviewer: Similar feedback as above:

```suggestion
# Adjusting quality goals

Adjust the **quality goals** to help you monitor the progress of the code quality in your repository dashboard, and configure which files Codacy considers complex or duplicated.
```

To access the quality goals, open your repository **Settings**, tab **Goals**. The following screenshot displays the default configuration values:

![Quality goals](images/quality-settings-goals.png)

-   **Issues are over:** Defines the threshold displayed on the tab **Issues** of the quality evolution chart.
-   **Complexity is over:** Defines the threshold displayed on the tab **Complexity** of the quality evolution chart.
-   **File is complex when over:** A file is considered complex when its complexity is over this value.
-   **Duplication is over:** Defines the threshold displayed on the tab **Duplication** of the quality evolution chart.
-   **File is duplicate when over:** A file is considered duplicated when it has more clones than this value.
-   **Coverage is under:** Defines the threshold displayed on the tab **Coverage** of the quality evolution chart.

## See also

-   [Which metrics does Codacy calculate?](../faq/code-analysis/which-metrics-does-codacy-calculate.md)

> -   [Why does Codacy show unexpected coverage changes?](../faq/code-analysis/why-does-codacy-show-unexpected-coverage-changes.md)

Reviewer: The information behind this link is only relevant in the context of pull requests, and only the quality gates have an influence on pull requests.

So, for goals it doesn't make sense to include this link:

```suggestion
```

Author: Right! I included it because it was part of the previous note box that we applied to both Gates and Goals... 🤔
